### PR TITLE
Minimale home page sign buttons

### PR DESCRIPTION
**Pull request by abg5f**

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -14,20 +14,18 @@
         Créez un carnet de bord digital pour votre nouveau petit héro !</p>
```
Comment (RonanGir) [inline app/views/pages/home.html.erb:14]: It missing a paragraph tag : 
 <p> Créez un carnet de bord digital pour votre nouveau petit héro  ! </p>

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -14,20 +14,18 @@
...
         <br>
```
Comment (RonanGir) [inline app/views/pages/home.html.erb:15]: Why 'br' tag  ? Is it not preferable with margin / paddin bottom ?

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -14,20 +14,18 @@
...
         <br>
-        <a class="btn btn-danger" href="#features"> En savoir plus</a>
+          <div class="sign-buttons">
+            <%= link_to "M'identifier", new_user_session_path, class: "btn btn-lg btn-success" %>
+            <%= link_to "M'enregistrer", new_user_registration_path, class: "btn btn-lg btn-primary text-green"%>
+          </div>
+        <div class="see-more">
+<!--           <a class="btn btn-danger" href="#features"> En savoir plus</a> -->
```
Comment (RonanGir) [inline app/views/pages/home.html.erb:22]: Is it useful to keep it commented ?

**Comment by RonanGir** [comment]
I added some review in the code. Please check and tell me what do you think about ?

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (abg5f) [reply app/views/pages/home.html.erb:14]: Hello, i fixed it ! Thank you :)

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (abg5f) [reply app/views/pages/home.html.erb:15]: I'll fix it ASAP, it's an easy way and it's more quickly, that's why. But it's a bit painfull yea.

File: app/views/pages/home.html.erb (hunk 3 above)
Comment (abg5f) [reply app/views/pages/home.html.erb:22]: It's fixed :+1: Thanks

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (RonanGir) [reply app/views/pages/home.html.erb:15]: Completely understand. Will check everything together. (must be cool)

**Pull request merged**